src/android/config.js

```diff
@@ -1,92 +1,285 @@
+    return (
+      <div>
+        <div className={classes.title}>
+          {new Date(record.pushDate).format("yyyy-MM-dd")}
+        </div>
+        <div>
           <span
-              style={{ backgroundColor: series.style.color }}
-              className={classes.tooltipKey}
+            style={{ backgroundColor: series.style.color }}
+            className={classes.tooltipKey}
           />
-                    {series.label}
-                    {' '}:
-                    {round(record.result, { places: 3 })}
-                </div>
-                <div>
-                    {(() => {
-                        const diffMax = record.result - referenceRange.max;
-                        const diffMin = record.result - referenceRange.min;
+          {series.label} :{round(record.result, { places: 3 })}
+        </div>
+        <div>
+          {(() => {
+            const diffMax = record.result - referenceRange.max;
+            const diffMin = record.result - referenceRange.min;
 
-                        if (diffMax > 0) {
-                            return (
-                                <span className={classes.above}>
+            if (diffMax > 0) {
+              return (
+                <span className={classes.above}>
                   {`${round(diffMax, {
-                      places: 2,
+                    places: 2
                   })}ms above target`}
                 </span>
-                            );
-                        }
-                        if (diffMin < 0 ){
-                            return (
-                                <span className={classes.below}>
-                {`${round(-diffMin, {
-                    places: 2,
-                })}ms below target`}
-              </span>
-                            );
-                        }
+              );
+            }
+            if (diffMin < 0) {
+              return (
+                <span className={classes.below}>
+                  {`${round(-diffMin, {
+                    places: 2
+                  })}ms below target`}
+                </span>
+              );
+            }
 
-                        return (
-                            <span className={classes.between}>meets target</span>
-                        );
+            return <span className={classes.between}>meets target</span>;
+          })()}
+        </div>
+      </div>
+    );
+  }
+);
 
+const PLATFORMS = [
+  {
+    id: "g5",
+    label: "Moto G5 (arm7)",
+    filter: {
+      eq: {
+        platform: "android-hw-g5-7-0-arm7-api-16"
+      }
+    }
+  },
+  {
+    id: "p2-aarch64",
+    label: "Pixel 2 (aarch64)",
+    filter: {
+      eq: {
+        options: "opt",
+        platform: "android-hw-p2-8-0-android-aarch64"
+      }
+    }
+  }
+];
 
-                    })()}
-                </div>
-            </div>
-        );
-    },
-);
+const COMBO_TABLE = {
+  header: ["browser", "browserLabel", "suite", "suiteLabel", "filter"],
+  data: [
```

Comment:
I am very sorry you are doing this header/data thing, with some conversion at the end.  Long ago these files were formatted like tables, but then the formatting rules got a hold of them.

Reply:
Oh, now I know the reason why :)
I have updated the code following your pointers. It would be cool if you could take a look again.
@klahnakoski